app/models/studio.rb

```diff
@@ -1,3 +1,8 @@
 class Studio < ApplicationRecord
   has_many :movies
+  has_many :actors, through: :movies
+
+  # def self.unique_actors
+  #   where()
```

Comment:
First - good job making this extra association! This helps a lot!! 

You had the right idea in making this a method instead of doing `.uniq` in the controller - so close! Try refactoring this piece to use this method instead, but maybe it doesn't need to be a class method....? 

If it's not a class method, then we do have access to the `actors` association in this Studio - some food for thought.